pom.xml

```diff
@@ -66,7 +66,7 @@
         <dependency>
             <groupId>com.palmergames.bukkit.towny</groupId>
             <artifactId>Towny</artifactId>
-            <version>0.96.0.0</version>
+            <version>0.96.1.3</version>
```

Comment:
- I update this to the earliest towny release version which supports the dnc feature.